templates/registration/includes/profile_stats.html

```diff
@@ -0,0 +1,56 @@
+{% load staticfiles %}
+{% load humanize %}
+
+<div class="investments">
+    <h2>EARNINGS</h2>
+    <div class="break">
+        <hr>
+    </div>
+</div>
+<h1 id="percentage"></h1>
+<h4 style="color: #4cbd7e;""><i class="fa fa-arrow-up" aria-hidden="true"></i>${{ total_earned|floatformat:2|intcomma }} | <span style="color: #4d96ad">${{ total_investments|floatformat:2|intcomma }}</span></h4>
+<h3 class="roi">Return on Investment</h3>
+
+<div class="profile-container">
+    <canvas id="myChart"></canvas>
+</div>
+
+<div>
+    {% for artist in artists.values %}
+        <div class="profile-stats">
+            {% if artist.total_earned %}
+                <p>{{ artist.name }}</p>
+                <p style="color: #4cbd7e;""><i class="fa fa-arrow-up" aria-hidden="true"></i>${{ artist.total_earned|floatformat:2|intcomma }}</p>
```

Comment:
You have two quotes beside each other here again. Ironically, this corrects the issue up above.